gqcp/tests/Mathematical/Representation/Matrix_test.cpp

```diff
@@ -91,6 +91,73 @@ BOOST_AUTO_TEST_CASE(Matrix_FromFile) {
 }
 
 
+BOOST_AUTO_TEST_CASE(areEqualEigenvectors) {
+
+    // Test areEqualEigenvectors with an example.
+    GQCP::VectorX<double> a {3};
+    GQCP::VectorX<double> b {3};
+    GQCP::VectorX<double> c {3};
+    GQCP::VectorX<double> d {3};
+
+    a << 2, 3, 1;
+    b << 2, 3, 1;
+    c << -2, -3, -1;
+    d << 2, 3, 0;
```

Comment:
If I remember correctly, you can use the brace initializer here.

```suggestion
    GQCP::VectorX<double> a {2, 3, 1};
    GQCP::VectorX<double> b {2, 3, 1};
    GQCP::VectorX<double> c {-2, -3, -1};
    GQCP::VectorX<double> d {2, 3, 0};
```

Maybe you'll have to specify the type `GQCP::Vector<double, 3>`.